tests/test_run_reference_vmm.py

```diff
@@ -214,12 +253,21 @@ def test_reference_vmm_with_disk(kernel, disk):
 
     vmm_process, tmp_disk_path = start_vmm_process(kernel, disk_path=disk)
 
-    expect_string(vmm_process, "Ubuntu 20.04 LTS ubuntu-rust-vmm ttyS0")
+    prompt = 'root@ubuntu-rust-vmm:~#'
+    expect_string(vmm_process, prompt)
+
+    cmd = 'lsblk --json'
+    output = run_cmd_inside_vm(cmd.encode(), vmm_process, prompt.encode(), timeout=5)
```

Comment:
Waiting for 5 seconds seems like too much. The output of `lsblk` should come instantly.

Reply:
Okay. I should change this to may be 1 sec. So for whatever reasons if `lsblk --json` does not return in 1 seconds that can be treated as failure right? I will also have to modify the `time.sleep` below to use something like `0.2` seconds, so that  we don't  get false positive exceptions.